### DISCUSSION TITLE
什麼是人工智慧？

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#200] [General]
https://www.netapp.com/zh-hant/artificial-intelligence/what-is-artificial-intelligence/
人工智慧 (AI) 是透過建立及應用內建於動態運算環境中的演算法，來模擬人類智慧過程的基礎。簡言之， AI 的目標是試圖讓電腦像人類一樣思考和行動。

實現這項目標需要三個關鍵要素：

運算系統
資料與資料管理
進階 AI 演算法 (程式碼)
期望結果越接近人類，對資料量和處理能力的要求越高。

**Comment by cobb110202008** [comment]
人工智慧的發展已經有一段時間了，但這種根本性的問題仍舊值得我們去思考及探究。

**Comment by wh20502** [comment]
文中提到詐騙偵測，我覺得現在這個時間非常重要希望趕快開發，whoscall已經擋掉大部分了，但是每次電話響都要害怕到底是不是詐騙或是推銷投資，害我非常兩難...

**Comment by 110409538** [comment]
人工智慧是未來世界的主流。

**Comment by Shiba010** [comment]
人工智慧儼然成為了現代以及未來的趨勢。

**Comment by ghost** [comment]
期待AI在未來是對人類很好的 而不是拿來傷害人民

**Comment by lx3211** [comment]
最近很紅的chatGPT，有人向他詢問說如果他不會有任何責任的話，他最想做什麼，結果他回答了想要消滅人類，雖然很有可能這就是人類想要他回答的答案（各種電影等資料）

**Comment by limitx0** [comment]
期待未來 AI 能幫人做更多事

**Comment by Ben6uo** [comment]
人工智慧是一種科技，它使計算機具有某些人类的智能行為，例如解决问题、做决策、學習、解决推理等。人工智慧可以分為硬件方面和軟件方面，硬件方面包括計算機硬件，軟件方面包括用於模擬人類智能的程序。
以上是用人工智慧打出來的

**Comment by chia-kai-chang** [comment]
從現在起，我們的小機器人會幫我們判斷，文章的資訊量是否充足，是不是原創文章喔。
如果有看到「NCUEDU-AI」按的眼睛圖案，就代表資訊量不足，暫時不予計點喔~
模型會定期更新，理論上會越做越準的。總之，就是想要鼓勵大家多多用文字闡述自己的觀點~ 即使不是主流，也很歡迎，因為那是你自己的觀點~

**Comment by boson13579** [comment]
AI只能說是統整人類思考後寫出的產物並對其進行模擬，並不能像人類一樣有自己的思考，也比較難構想出創新的點子，只能等到足夠多的樣本出現後才有AI的立足處。
